suites/apis/centralizedAuth.js

```diff
@@ -0,0 +1,655 @@
+     RBAC policy information is there
+        > NOTE: there is an existing test that might satisfy this with a few updates
+
+4) - Create some records in indexd with AND logic, have CLIENTB try to create signed
+     url for file for user that does NOT have necessary permissions, ensure failure.
+
+   - Same as above but user DOES have permissions. Ensure successfull signed URL that
+     we can access data with
+
+
+We probably want some tests to make sure that after usersyncing and changing a user's
+policies, they can no longer create signed urls?
+
+dbgap syncing tests would be good too (since all these rely on user.yaml)
+
+------------------------------------
+Default Authorization Details
+------------------------------------
+
+Access details from defualt usersync:
+
+Create CLIENT with `abc-admin` and `gen3-admin` policy (CRUD under /abc and /gen3)
+Create CLIENTB with `abc-admin` policy (CRUD under /abc)
+
+cdis.autotest@gmail.com (mainAcct)
+    - abc-admin
+
+dummy-one@planx-pla.net (auxAcct1)
+    - abc.programs.test_program.projects.test_project1-viewer
+
+smarty-two@planx-pla.net (auxAcct2)
+    - abc.programs.test_program2.projects.test_project3-viewer
+
+dcf-integration-test-0@planx-pla.net (user0)
+    - gen3-admin
+*/
+const { Commons } = require('../../utils/commons.js');
+const chai = require('chai');
+const { Bash } = require('../../utils/bash.js');
+const apiUtil = require('../../utils/apiUtil.js');
+const uuid = require('uuid');
+
+const bash = new Bash();
+
+const fs = require('fs');
+
+const I = actor();
+
+/*
+for twoProjectsFile, use cdis.autotest@gmail.com
+
+Each has ONE of these policies:
+    dummy-one@planx-pla.net (auxAcct1)
+        - abc.programs.test_program.projects.test_project1-viewer
+    smarty-two@planx-pla.net (auxAcct2)
+        - abc.programs.test_program2.projects.test_project3-viewer
+*/
+const indexed_files = {
+  abcFooBarFile: {
+    filename: 'testdata',
+    link: 's3://cdis-presigned-url-test/testdata',
+    md5: '73d643ec3f4beb9020eef0beed440ad0',
+    acl: [],
+    rbac: ['/abc/programs/foo/projects/bar'],
+    size: 9,
+  },
+  gen3TestTestFile: {
+    filename: 'testdata',
+    link: 's3://cdis-presigned-url-test/testdata',
+    md5: '73d643ec3f4beb9020eef0beed440ad1',
+    acl: [],
+    rbac: ['/gen3/programs/test/projects/test'],
+    size: 10,
+  },
+  twoProjectsFile: {
+    filename: 'testdata',
+    link: 's3://cdis-presigned-url-test/testdata',
+    md5: '73d643ec3f4beb9020eef0beed440ad2',
+    acl: [],
+    rbac: [
+      '/abc/programs/test_program/projects/test_project1',
+      '/abc/programs/test_program2/projects/test_project3'
+    ],
+    size: 11,
+  }
+}
+
+let new_gen3_records = {
+  fooBarFile: {
+    filename: 'testdata',
+    link: 's3://cdis-presigned-url-test/testdata',
+    md5: '73d643ec3f4beb9020eef0beed440ad4',
+    acl: [],
+    rbac: ['/gen3/programs/test/projects/test'],
+    size: 9,
+  },
+  deleteMe: {
```

Comment:
for indexd DELETE, you only need `did` and `rev`:
- i think you don't need to keep so much information in your `deleteMe` files
- maybe i missed it but i don't see where you create the `deleteMe` files in indexd, and i don't think deletion tests will work if you don't create the records first... you should use `addFileIndices` on these files in the `Before` block, it will populate `file.rev`

Reply:
these get created in the test itself: https://github.com/uc-cdis/gen3-qa/blob/5ea07c7d23e5b4143091d481a182ee4c18f3cf0b/suites/apis/centralizedAuth.js#L213 when it adds the other files too